nav2_regulated_pure_pursuit_controller/src/regulated_pure_pursuit_controller.cpp

```diff
@@ -176,6 +192,11 @@ void RegulatedPurePursuitController::configure(
   collision_checker_ = std::make_unique<nav2_costmap_2d::
       FootprintCollisionChecker<nav2_costmap_2d::Costmap2D *>>(costmap_);
   collision_checker_->setCostmap(costmap_);
+
+  distance_profile_ = new ruckig::Ruckig<1>{control_duration_};
```

Comment:
Use unique_ptrs, we've removed any raw memory management  from Nav2

Reply:
Switched to smart pointers.